### DISCUSSION TITLE
Alias Group to API Object

### DISCUSSION THREAD

**Discussion by casualmatt** [directus/directus#11587] [Feature Requests]
When in directus we specify a Group(alias) of element, this is not represented in the API.
![image](https://user-images.githubusercontent.com/85650530/153585424-79c11c67-b574-48ba-8f0d-50cdc76702ee.png)

Like in this case, the object in the api should be:

```
{
  "id":"612da06c-10f9-4a91-a48c-b0f9cd9b1252",
  "country":{
    "name":"switzerland",
    "flag":"https://URL"
  }
}
```

or to not mess up old install, let it be an option when creating that kind of alias field, to be visible in the api too.

**Comment by rijkvanzanten** [comment]
> Like in this case, the object in the api should be:

That sounds like an opinion 😁 

I agree there are some nice use cases for allowing grouped API outputs, but I wouldn't go as far as to say that is the correct way and how it should be for everybody for every group 🙂 (for example it would get really annoying when you have nested groups for collapsable sections for example)

**Reply by casualmatt** [reply]
Yes, that's why i think is more feasible to respect old install to add it as an option.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
